docs/user-guide/scaling.md

```diff
@@ -0,0 +1,101 @@
+# Scaling
+
+Cubed is specifically designed to maintain optimal performance while processing large arrays (of order Terabytes so far).
+This page aims to provide a deeper understanding of how Cubed's design scales in both theoretical and practical scenarios.
+
+## Preface: Types of Scaling
+
+There are different types of scaling to consider in distributed computing:
+
+Horizontal versus Vertical Scaling: Horizontal scaling refers to adding more machines to the system to improve its throughput, while vertical scaling means enhancing the speed & resources of an existing machine.
```

Comment:
"enhancing the speed & resources of an existing machine" - or replacing an existing machine with a larger one with more resources.